example/main.tf

```diff
@@ -1,3 +1,7 @@
+terraform {
+  required_version = "0.15.4"
```

Comment:
Just double checking here. In the PR description you mentioned terraform 0.14, but here we are requiring 0.15, is it ok?

Reply:
Yep, any version starting from 0.14. So 0.15 fits the requirement